localstripe/resources.py

```diff
@@ -542,32 +542,54 @@ class Coupon(StripeObject):
 
     def __init__(self, id=None, name=None, duration=None, amount_off=None,
                  percent_off=None, currency=None, metadata=None,
-                 duration_in_months=None, **kwargs):
+                 duration_in_months=None, max_redemptions=None, redeem_by=None, **kwargs):
         if kwargs:
             raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
 
         amount_off = try_convert_to_int(amount_off)
-        percent_off = try_convert_to_int(percent_off)
+        percent_off = try_convert_to_float(percent_off)
```

Comment:
https://stripe.com/docs/api/coupons/object#coupon_object-percent_off documentation says percent_off is decimal